src/DurableFunctions.FSharp/Activity.fs

```diff
@@ -66,6 +66,19 @@ module Activity =
                 r
         c.CallActivityWithRetryAsync<'b> (activity.name, options, arg)
 
+    /// Call the activity by name passing an object as its input argument
+    /// and specifying the type to expect for the activity output. Apply retry
+    /// policy in case of call failure(s).
+    let callByNameWithRetries<'a> (policy: RetryPolicy) (name:string) arg (c: DurableOrchestrationContext) =
+        let options = 
+            match policy with
+            | ExponentialBackOff e ->
+                        let r = RetryOptions(firstRetryInterval = e.FirstRetryInterval,
+                                                maxNumberOfAttempts = e.MaxNumberOfAttempts)
+                        r.BackoffCoefficient <- e.BackoffCoefficient
+                        r
```

Comment:
Maybe, add a helper function to calculate `options` instead of copy-pasting. With more policies, this might get out-of-sync in the future.